client/status/status.go

```diff
@@ -6,39 +6,10 @@ package apistatus
 //   - statuses that implement the build-in error interface are considered failed statuses;
 //   - all other value types are considered successes (nil is a default success).
 //
-// In Go code type of success can be determined by a type switch, failure - by a switch with errors.As calls.
+// In Go code type of success can be determined by a type switch, failure - by a switch with [errors.As] calls.
 // Nil should be considered as a success, and default switch section - as an unrecognized Status.
 //
-// To convert statuses into errors and vice versa, use functions ErrToStatus and ErrFromStatus, respectively.
-// ErrFromStatus function returns nil for successful statuses. However, to simplify the check of statuses for success,
-// IsSuccessful function should be used (try to avoid nil comparison).
 // It should be noted that using direct typecasting is not a compatible approach.
 //
-// To transport statuses using the NeoFS API V2 protocol, see StatusV2 interface and FromStatusV2 and ToStatusV2 functions.
+// To transport statuses using the NeoFS API V2 protocol, see [StatusV2] interface and [FromStatusV2] and [ToStatusV2] functions.
 type Status any
```

Comment:
I'm not sure what to do with this `Status`. We may completely remove it and replace with `any` all usages in the code. Another option is try to make extended interface/something to save the context in the code, I mean `status apistatus.Status` means more then `status any`

Reply:
We can remove it and convert errors to/from statusv2 directly, this type makes zero sense to me, it's not really used anywhere outside of this package.

Reply:
Removed